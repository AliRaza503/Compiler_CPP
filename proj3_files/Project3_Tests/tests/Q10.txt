{
    while(a){
        b = * c a;
    }
}