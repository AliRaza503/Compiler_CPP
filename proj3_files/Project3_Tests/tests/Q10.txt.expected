a: bool #
b: bool #
c: bool #
